test/exclude-base-props.test.js

```diff
@@ -0,0 +1,37 @@
+// Copyright IBM Corp. 2013,2016. All Rights Reserved.
+// Node module: loopback-datasource-juggler
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+// This test written in mocha+should.js
+'use strict';
+var should = require('./init.js');
+var assert = require('assert');
+
+var jdb = require('../');
+var ModelBuilder = jdb.ModelBuilder;
+
+describe('exclude properties ', function() {
+  it('from base model', function(done) {
+    var ds = new ModelBuilder();
+    // this excludes id property from 'base: Model' We still need to pass in idInjection: false since User model tries to
+    // add id again to the model.
+    var User = ds.define('User', {name: String, password: String},
+      {idInjection: false, excludeBaseProperties: ['id']});
+    // User will have these properties: name, password
+    var properties = User.definition.properties;
+    assert(('name', 'password' in properties));
+    // id should not be found in the properties list
+    assert(!('id' in properties));
+    // this excludes id property from the base model and and password property coming from base 'User' model since customer is
+    // extended from User.
+    var Customer = User.extend('Customer', {vip: {type: String}},
+      {idInjection: false, excludeBaseProperties: ['password']});
+    // Customer will have these properties: name(from UserModel) & vip
+    properties = Customer.definition.properties;
+    assert(('name', 'vip' in properties));
```

Comment:
It should be `assert( ('name' in properties) && ('vip' in properties) )`. See https://nodejs.org/api/assert.html#assert_assert_value_message.

It's also recommended to have an error message as the 2nd argument so that it fails with a reason. Or even better to use `chai` for expectation.